WindowsServerDocs/identity/ad-fs/technical-reference/The-Role-of-Attribute-Stores.md

```diff
@@ -17,20 +17,20 @@ For more information about how claims are generated, see [The Role of Claims](Th
 ## How attribute stores fit in with your AD FS deployment goals
 The location of the user attribute store and the location from which users authenticate determine how you design AD FS to support the user identities. Depending on where the attribute store is located and where users will access the application \(in an intranet or on the Internet\), you can use one of the following deployment goals:
 
--   [Provide Your Active Directory Users Access to Your Claims-Aware Applications and Services](/previous-versions/windows/it-pro/windows-server-2012-R2-and-2012/dd807071(v=ws.11))—In this goal, users in your organization access an AD FS–secured application or service \(either your own application or service or a partner's application or service\) when the users are logged on to Active Directory in the corporate intranet.
+-   [Provide Your Active Directory Users Access to Your Claims-Aware Applications and Services] — In this goal, users in your organization access an AD FS–secured application or service \(either your own application or service or a partner's application or service\) when the users are logged on to Active Directory in the corporate intranet.
```

Comment:
To remove the link is fine, but you need to also remove the brackets around the descriptor text. So it should just look like:  "Provide Your Active Directory Users Access to Your Claims-Aware Applications and Services  — ". If you can make that change to all of these, I'll go ahead and publish.

Reply:
Sorry for not being aware in my previous comment below, I did not notice that the comment from eross-msft is just a comment, and not a change suggestion. Please accept my suggestion for the change with a button to merge/commit the change:
```suggestion
- Provide Your Active Directory Users Access to Your Claims-Aware Applications and Services — In this goal, users in your organization access an AD FS–secured application or service (either your own application or service or a partner's application or service) when the users are logged on to Active Directory in the corporate intranet.
```
Also note: the backslash escape character is usually not needed before regular parenthesis characters.